examples/networking/http_filter/http-parse-complete.py

```diff
@@ -93,206 +97,221 @@ def help():
     print("    http-parse -i wlan0     # bind socket to wlan0")
     exit()
 
-#arguments
-interface="eth0"
+
+# arguments
+interface = "eth0"
 
 if len(argv) == 2:
-  if str(argv[1]) == '-h':
-    help()
-  else:
-    usage()
+    if str(argv[1]) == "-h":
+        help()
+    else:
+        usage()
 
 if len(argv) == 3:
-  if str(argv[1]) == '-i':
-    interface = argv[2]
-  else:
-    usage()
+    if str(argv[1]) == "-i":
+        interface = argv[2]
+    else:
+        usage()
 
 if len(argv) > 3:
-  usage()
+    usage()
 
-print ("binding socket to '%s'" % interface)
+print("binding socket to '%s'" % interface)
 
 # initialize BPF - load source code from http-parse-complete.c
-bpf = BPF(src_file = "http-parse-complete.c",debug = 0)
+bpf = BPF(src_file="http-parse-complete.c", debug=0)
 
-#load eBPF program http_filter of type SOCKET_FILTER into the kernel eBPF vm
-#more info about eBPF program types
-#http://man7.org/linux/man-pages/man2/bpf.2.html
+# load eBPF program http_filter of type SOCKET_FILTER into the kernel eBPF vm
+# more info about eBPF program types
+# http://man7.org/linux/man-pages/man2/bpf.2.html
 function_http_filter = bpf.load_func("http_filter", BPF.SOCKET_FILTER)
 
-#create raw socket, bind it to interface
-#attach bpf program to socket created
+# create raw socket, bind it to interface
+# attach bpf program to socket created
 BPF.attach_raw_socket(function_http_filter, interface)
 
-#get file descriptor of the socket previously created inside BPF.attach_raw_socket
+# get file descriptor of the socket previously created inside BPF.attach_raw_socket
 socket_fd = function_http_filter.sock
 
-#create python socket object, from the file descriptor
-sock = socket.fromfd(socket_fd,socket.PF_PACKET,socket.SOCK_RAW,socket.IPPROTO_IP)
-#set it as blocking socket
+# create python socket object, from the file descriptor
+sock = socket.fromfd(socket_fd, socket.PF_PACKET, socket.SOCK_RAW, socket.IPPROTO_IP)
+# set it as blocking socket
 sock.setblocking(True)
 
-#get pointer to bpf map of type hash
+# get pointer to bpf map of type hash
 bpf_sessions = bpf.get_table("sessions")
 
-#packets counter
+# packets counter
 packet_count = 0
 
-#dictionary containing association <key(ipsrc,ipdst,portsrc,portdst),payload_string>
-#if url is not entirely contained in only one packet, save the firt part of it in this local dict
-#when I find \r\n in a next pkt, append and print all the url
+# dictionary containing association <key(ipsrc,ipdst,portsrc,portdst),payload_string>
+# if url is not entirely contained in only one packet, save the firt part of it in this local dict
+# when I find \r\n in a next pkt, append and print all the url
 local_dictionary = {}
 
 while 1:
-  #retrieve raw packet from socket
```

Comment:
We might want to disable the rule on indentations too, at least in some subdirectories. Otherwise, the whole file is changed and it's just messing with git blame. It probably matters less for examples anyway.